### DISCUSSION TITLE
Espressif-IDE v3.0.0 — LSP Support for C/C++ Editor · Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#59] [General]
# Espressif-IDE v3.0.0 — LSP Support for C/C++ Editor · Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/espressif-ide-lsp-support/

**Comment by guoruoone** [comment]
how to use open declaration in LSP editor?

**Reply by kolipakakondal** [reply]
Ctrl+Click should work fine with the LSP Editor, However, if you see any issues, please check this [link](https://github.com/espressif/idf-eclipse-plugin/issues/995#issuecomment-2276935107). You can configure this in the preferences window.

**Comment by ioviedodev** [comment]
Hi, great information. Finally I undertand the change. It is look really cool the experiencie developing in Espressiff IDE with the last release 3.1.0, but I think that there are some issues still. When I control click I got this issue: (unknown results: indexer is not enabled)

**Reply by ioviedodev** [reply]
*sorry Ctrl + H no works, I got "unknown results: indexer is not enabled"
